dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/TestCommon/CommonTestCode.cs

```diff
@@ -0,0 +1,520 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if USE_VSTEST
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.ApplicationModel.Resources;
```

Comment:
Can these be moved outside the `#if` since they are common to both?
```
using System;
using Microsoft.ApplicationModel.Resources;
```

Reply:
for some reason, i feel this is cleaner.

Reply:
sure, if that's what you prefer. :)